src/SLCore/Listener/Logger/LogParams.cs

```diff
@@ -26,6 +26,12 @@ namespace SonarLint.VisualStudio.SLCore.Listener.Logger;
 public class LogParams
 {
     public string message;
+    // nullable
+    public string configScopeId;
+    public string threadName;
+    public string loggerName;
+    // nullable
+    public string stackTrace;
```

Comment:
We are missing the `loggedAt` property. Even if we don't use it for now, I think we should add it.